vpath/pjtester.yaml

```diff
@@ -0,0 +1,6 @@
+pjNames:
+  - pjName: "pre-main-kyma-integration-k3s-central-app-gateway-compass"
```

Comment:
Should it be eventually removed?

Reply:
Yes, It's here just for the testing purposes